javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+const $form = document.forms['contact-form'];
+
+function submitFunction(event) {
+  const newObject = {};
+  newObject.name = $form.elements.name.value;
+  newObject.email = $form.elements.email.value;
+  newObject.message = $form.elements.message.value;
+  event.preventDefault();
```

Comment:
This should be at the top of the function, because it's the first thing we want to run